daisy_workflows/build-publish/enterprise_linux/rocky_linux_8_optimized_gcp_nvidia_550.wf.json

```diff
@@ -1,5 +1,5 @@
 {
-  "Name": "rocky-linux-8-optimized-gcp-nvidia-550",
+  "Name": "rocky-linux-8-optimized-gcp-nvidia-570",
```

Comment:
This is the 550 workflow. Shouldn't this always be 550?

You likely means to copy the 550.wf file to 570.wf and then update that.

Reply:
Indeed this was a copypaste issue. Resolved.